plugins/StereoControl/StereoControl.cpp

```diff
@@ -0,0 +1,444 @@
+/*
+ * StereoControl.cpp
+ *
+ * Copyright (c) 2020 Lost Robot <r94231@gmail.com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#include "StereoControl.h"
+
+#include "embed.h"
+#include "interpolation.h"
+#include "lmms_math.h"
+#include "plugin_export.h"
+
+extern "C"
+{
+
+Plugin::Descriptor PLUGIN_EXPORT stereocontrol_plugin_descriptor =
+{
+	STRINGIFY(PLUGIN_NAME),
+	"Stereo Control",
+	QT_TRANSLATE_NOOP("pluginBrowser", "General utility and stereo processing, including multiple types of panning and stereo enhancement."),
```

Comment:
`pluginBrowser` -> `PluginBrowser`, the same for your plugins. Existing plugins are already fixed in `master`, FYI.